docs/getting_started/components.md

```diff
@@ -23,8 +23,12 @@ View the following table for a description of the {{site.data.keyword.ieam}} com
 
 |Component|Version|Description|
 |---------|-------|----|
-|Cloud Sync Service |1.4.1-173|The CSS is the component of the Model Management System (MMS) that stores machine learning models for later deployment to edge devices.|
-|Exchange API|2.54.0-176|The Exchange provides the REST API that holds all the definitions (patterns, policies, services, and so on) used by all the other components in {{site.data.keyword.ieam}}.|
-|Secure Device Onboard|1.8.6-83|The SDO component enables technology created by Intel to make it easy and secure to configure edge devices and associate them with an edge management hub.|
-|Cluster Agent|2.27.0-173|This is a container image which is installed as an agent on edge clusters to enable node workload management by {{site.data.keyword.ieam}}.|
-|Device Agent|2.27.0-173|This component is installed on edge devices to enable node workload management by {{site.data.keyword.ieam}}.|
+|[IBM Cloud Platform Common Services](https://www.ibm.com/docs/en/cpfs)|3.6.x|This is a set of foundational components that are installed automatically as part of the {{site.data.keyword.ieam}} operator installation.|
```

Comment:
Don't think this section applies to Open Horizon

Reply:
I reverted the IBM Cloud Platform reference in this commit
https://github.com/open-horizon/open-horizon.github.io/pull/220/commits/4ba6cadd4e2544148e7a2f33610d7a9245a37db2